### ISSUE TITLE
Avoid to restart DM if previously started when RabbitMQ is down

### ISSUE THREAD

**Issue by cdeneux** [roboconf/roboconf-platform#362]
If you start Roboconf DM when the messaging system is not started, no error is displayed, expected in Roboconf log file) about messaging system down.
Next, if you try to deploy an application, this does not work. It is required to restart the Roboconf DM.

In a user point if view, it will be more user friendly to have not to restart the DM but only:
* start messaging system,
* the DM tries an automatic reconnection
and to have, on DM startup, an error message when its configuration is not valid

Perhaps, the Roboconf console (webapp) should not be available until the messaging system is up.

**Comment by vincent-zurczak** [comment]
Duplicate of #288

**Comment by vincent-zurczak** [comment]
#202 is not the solution.
Auto-reconnection works only once a successful connection was established.

This kind error can happen when the messaging server was started after the DM. And it can happen upon an invalid reconfiguration (e.g. invalid credentials). This is clearly a user error. We can do two things about it...

* Highlight such errors in the web console.
* Provide a Karaf command (so, in Karaf's command line) to force the DM to reload its configuration. This command will also be necessary on agents. Karaf does not natively provides such a solution. `touch my.cfg` does nothing and it would be quite complicated to hack our iPojo configuration as we already make some checks about configuration changes. This command will be less radical than `bundle:refresh`.

I do not think providing an automatic reconnection would be a good idea. Not when the DM or agents start (#202 covers such cases but after a successful initial connection). Failing at start time may be an error in boot order, but also an invalid configuration. We need to focus on giving feedback to the user.

**Comment by cdeneux** [comment]
In this use case, it is sufficient that the Roboconf DM startup fails with an error message. No need to have a configuration reload or highlights in web console. As simple user, I can re-launch the DM after correcting configuration or starting RabbitMQ.

**Comment by vincent-zurczak** [comment]
`.karaf` only launches Karaf, which indirectly launches the DM.
`.karaf` cannot fail if the DM's configuration is invalid. But we could display an error message in the console in such a situation, yes.

**Comment by vincent-zurczak** [comment]
This issue will be solved through these 3 ones: #595 and #594.
We do not consider the web administration for the moment.

**Comment by vincent-zurczak** [comment]
Both sub-issues were solved.
